.github/workflows/pre-submit.yaml

```diff
@@ -0,0 +1,25 @@
+name: Go
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.15
+
+    - name: Build
+      run: make fmt vet
```

Comment:
@rgolangh 
The name is `Build` but this only runs `fmt` and `vet`? where's the actual build?